run-ci-live-tests.sh

```diff
@@ -21,6 +21,8 @@ DEX_PROC=$!
 FETCH_PROC=$!
 ./run src/tests/transaction-flow.ts --bundle | add_prefix "TRANSACTION_FLOW" &
 TRANSACTION_FLOW_PROC=$!
+./run src/examples/reducer/run-live.ts --bundle | add_prefix "REDUCER" &
+REDUCER_FLOW_PROC=$!
```

Comment:
New reducer test in CI handles some action serialization cases using lightnet graphql API.  The coverage on this is not ultra high, but it's a starting point, and covers the issue that zk noid originally isolated in their repo.